### PR TITLE
Fix Rydberg Formula, mv folders to Espanol to run intro and Dwarf

### PR DESCRIPTION
**Pull request by jsuarez314**
Intro (en & es)
I fixed the Rydberg formula.
Modify the Omega_Lambda values to 50% -> 0.5   and 75% -> 0.75

Dwarf (es)
Fixed some typos

Both (Intro and Dwarf)
I was checking the Español version, and some notebooks did not run because some libraries was not in the same path (tools, dat, student_andes), I made copies of these folders in the Espanol folder.

### CODE REVIEW

**Comment by alxogm** [comment]
Hi @jsuarez314, thanks for the work! I have one question, the runs that failed for you where done in binder or in your local copy? I think I've run all notebooks before without issues, so I'll check why its failing...

File: Espanol/Intro_es.ipynb [hunk 1]
```diff
@@ -574,8 +574,8 @@
     "pl.plot(dat['z'], dat['Effective magnitude'], marker='.', lw=0.0)\n",
     "\n",
     "pl.plot(zs, effmag(zs, 0.0, 6.), c='k',   label='No Dark Energy', alpha=0.5)\n",
-    "pl.plot(zs, effmag(zs, 0.7, 6.), c='k',   label='50% Dark Energy!')\n",
-    "pl.plot(zs, effmag(zs, 0.751, 6.), c='c', label='75% Dark Energy!')\n",
+    "pl.plot(zs, effmag(zs, 0.5, 6.), c='k',   label='50% Dark Energy!')\n",
```
Comment (michaelJwilson) [inline Espanol/Intro_es.ipynb:577]: Was this also wrong in the English?

Comment (jsuarez314) [reply Espanol/Intro_es.ipynb:577]: Yes, but no I do not remember if I fixed this on english version :(

**Comment by michaelJwilson** [comment]
Great catch on the Rydberg formula, thanks for this!  I dropped another initial comment about the English / Spanish consistency.  My impression was things had fallen behind a little. 

I don't want to be a pain, but given GitHub data limits, we need to rethink a structure that doesn't rely on copying all the data to the Espanol folder as you've done (at a quick glance).  E.g. with the drop-in READMEs.

The next few weeks, I'm a bit slammed but plan to get back onto this after.  Let's hold off on merging this until I get a good look, unless we split off the e.g. data duplicates etc.

**Comment by jsuarez314** [comment]
Hi @alxogm, I run the notebooks on binder. So, I first but not ellegant solution was to make copies of some folders in the Español version.

**Comment by michaelJwilson** [comment]
(I appreciate the hypocrisy on my part, given copying to Esp. has been the approach in the past).

**Pull request closed without merging**